xarray/coding/cftimeindex.py

```diff
@@ -261,6 +261,8 @@ def format_attrs(index, separator=", "):
         "length": f"{len(index)}",
         "calendar": f"'{index.calendar}'",
     }
+    if len(index) >= 3:
+        attrs["freq"] = f"'{index.freq}'"
```

Comment:
I decided to only show freq if that’s possible to calculate

Reply:
Pandas will show `freq=None` in that instance.  What do you think, should we do that here or no?

Reply:
then I will also add this here.